src/main/java/jenkins/branch/MultiBranchProject.java

```diff
@@ -1563,32 +1556,6 @@ public String getUrl() {
         }
 
         /**
-         * Returns a {@link hudson.model.Slave} on which this build was done.
-         *
-         * @return {@code null}, for example if the slave that this build run no longer exists.
-         */
-        @SuppressWarnings("unused") // used from jelly pages
-        @CheckForNull
-        public Node getBuiltOn() {
-            Jenkins j = Jenkins.getInstance();
-            if (builtOn == null || builtOn.isEmpty() || j == null) {
-                return j;
-            } else {
-                return j.getNode(builtOn);
-            }
-        }
-
-        /**
-         * Returns the name of the slave it was built on; null or "" if built by the master.
-         * (null happens when we read old record that didn't have this information.)
-         */
-        @Exported(name = "builtOn")
-        @SuppressWarnings("unused") // used by Jenkins API
-        public String getBuiltOnStr() {
-            return builtOn;
-        }
```

Comment:
Is there any risk that `getBuiltOn` and `getBuiltOnStr` were used somewhere else?